src/item/component/IconComponent.php

```diff
@@ -18,8 +18,8 @@ public function getName(): string {
 	public function getValue(): array {
 		return [
 			"textures" => [
-				"default" => $this->texture
-			]
+                'default' => $this->texture
```

Comment:
customies uses " and not ' everywhere in it's code, changing that makes no sense and is decreasing code quality (+ indentation is still tabulations)